app/models.py

```diff
@@ -137,5 +193,147 @@ class Config:
+                                    {"text": "Option 1"},
+                                    {"text": "Option 2"},
+                                    {"text": "Option 3"},
+                                ],
+                                "graded": False,
+                            },
+                            {
+                                "_id": "3039004",
+                                "text": "Which grade are you in?",
+                                "type": "multi-choice",
+                                "image": {
+                                    "url": "https://plio-prod-assets.s3.ap-south-1.amazonaws.com/images/afbxudrmbl.png",
+                                    "alt_text": "Image",
+                                },
+                                "options": [
+                                    {"text": "Option 1"},
+                                    {
+                                        "text": "Option 2",
+                                        "image": {
+                                            "url": "https://plio-prod-assets.s3.ap-south-1.amazonaws.com/images/afbxudrmbl.png"
+                                        },
+                                    },
+                                    {"text": "Option 3"},
+                                ],
+                                "correct_answer": [0, 2],
+                                "graded": True,
+                            },
+                            {
+                                "_id": "393920",
+                                "text": "Which subject are you studying?",
+                                "type": "multi-choice",
+                                "options": [
+                                    {"text": "Option 1"},
+                                    {"text": "Option 2"},
+                                    {"text": "Option 3"},
+                                ],
+                                "graded": False,
+                            },
+                        ],
+                    }
+                ],
+                "max_marks": 10,
+                "num_graded_questions": 3,
+                "metadata": {"quiz_type": "JEE", "subject": "Maths", "grade": "8"},
+            }
+        }
+
+
+class SessionAnswer(BaseModel):
+    """Model for the body of the request that creates a session answer"""
+
+    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
+    question_id: str
+    answer: Union[List[int], None] = None
+
+    class Config:
+        allow_population_by_field_name = True
+        arbitrary_types_allowed = True
+        json_encoders = {ObjectId: str}
+        schema_extra = {"example": {"question_id": "4567", "answer": [0, 1, 2]}}
+
+
+class UpdateSessionAnswer(BaseModel):
+    """Model for the body of the request that updates a session answer"""
+
+    answer: Union[List[int], None]
+
+    class Config:
+        schema_extra = {"example": {"answer": [0, 1, 2]}}
```

Comment:
specifying only `answer` here means that no one will be able to update anything other than the answer?
Like I can't update the session_id or user_id if I wanted to?